translations/de.ts

```diff
@@ -598,69 +579,42 @@ Soll der Proxyserver ignoriert werden und eine Direktverbindung hergestellt werd
 <context>
     <name>Core</name>
     <message>
-        <location filename="../src/core/core.cpp" line="260"/>
+        <location filename="../src/core/core.cpp" line="259"/>
         <source>Toxing on qTox</source>
-        <translation>Tox ist Toll!</translation>
+        <translation>Tox ist toll</translation>
```

Comment:
@alexander255 I personally disagree. "Tox ist toll!" sounds good. Anyway, it's an example description for the profile. It could be also "Tox macht Spaß!" oder "Viel Spaß mit Tox!", ... but actually: Sometimes it's better to not touch a running sentence - äh - system :).